new_images.yml

```diff
@@ -1,7 +1,7 @@
 ---
 new_images:
-  - spark: "3.1.1"
+  - spark: "3.0.1"
```

Comment:
Why are we downgrading it? Is it just for a specific release or this is for our newest spark container img?

Reply:
so it's a release for Spark 3.0(EMR 6.2.1)